doc/styles/CheckBox.md

```diff
@@ -76,7 +76,7 @@ IsDefaultStyle\*: Styles in this column will be set as the default implicit styl
 | `CheckBoxCheckBackgroundFillIndeterminatePointerOver`   | `SolidColorBrush` | `PrimaryBrush`                                                                                       |
 | `CheckBoxCheckBackgroundFillIndeterminatePressed`       | `SolidColorBrush` | `PrimaryBrush`                                                                                       |
 | `CheckBoxCheckBackgroundFillIndeterminateDisabled`      | `SolidColorBrush` | `OnSurfaceLowBrush`                                                                                  |
-| `CheckBoxFontFamily`                                    | `FontFamily`      | `MaterialMediumFontFamily`                                                                           |
+| `CheckBoxFontFamily`                                    | `FontFamily`      | `BodySmallFontFamily`                                                                                |
```

Comment:
@kazo0, @Xiaoy312 I just quickly checked the doc files and I have a little doubt and was wondering if there is any need to have more alignments. Some controls have FontFamily, FontWeight, etc... properties for Lightweight Styling and some others don't.

Reply:
Also @kazo0 I think we would need to do the same type of changes for Toolkit as well for some controls:
![image](https://github.com/unoplatform/Uno.Themes/assets/16295702/bb5549b3-c0df-4d61-b0d0-b1d4ae68723c)
https://github.com/unoplatform/uno.toolkit.ui/blob/476437c9511c28f0ce8c09c5ddd2cd79aae69cda/src/library/Uno.Toolkit.Material/Styles/Controls/v2/Chip.xaml#L378

Reply:
We'll need to discuss this further as we're kind of in a in-between state. The problem is what we think should happen if someone overrides something like `BodySmallFontFize/Family/Weight`. Should it be reflected in all controls or should you allow for more fine-grained control where you can override just the FontSize/Family/Weight scoped to a specific control. The way the staticresource aliasing works, we can't have both. I think we as well go for the global override route as that aligns with what is possible in Figma in terms of generating an app that overrides the semantic font resources at a global level and have it be reflected by all controls.

This would mean more breaking changes in the future though, since we would do away with things like `CheckBoxFontFamily`, `FilledButtonFontSize`, etc. and have them use `BodyLargeFontFamily`, `TitleSmallFontSize` directly.

cc: @carldebilly / @NVLudwig

Reply:
for now, this PR is just a stop-gap to stop using the `MaterialMedium/RegularFontFamily` directly, but more work will be needed

Reply:
Opened issue for this here: https://github.com/unoplatform/Uno.Themes/issues/1376